modules/retail/pages/retail-components.adoc

```diff
@@ -20,13 +20,19 @@ They are used to securely build operating system images.
 
 For more information on build hosts, see xref:administration:image-management.adoc[].
 
+
+
 == Branch Servers
 
 Branch servers should be physically located close to point-of-service terminals, such as in an individual store or branch office.
 Branch servers provide services for PXE boot, and act as an image cache, Salt broker, and proxy for software components (RPM packages).
 The branch server can also manage local networking, and provide DHCP and DNS services.
 
-
+[NOTE]
+====
+For monitoring, you can install Prometheus server on the Branch servers, and, for visualization and analysis, Grafana with multiple data sources on a dedicated host.
```

Comment:
I had to read the line multiple times to understand it. 😃 

Can it be split into two sentences, one for Prometheus and another for Grafana?

Reply:
Yes, I completely agree.  Thanks, Ornela, for the suggestion!